backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/queries.html.ftl

```diff
@@ -0,0 +1,322 @@
+           Show query content
+           </button>
+
+           <button class="btn btn-primary innerBtn" type="button" data-toggle="collapse" data-target="#error${queryCounter}" aria-expanded="false" aria-controls="error${queryCounter}" ${(data.error ? '' : 'disabled')}>
+           Show errors
+           </button>
+        </div>
+      </div>
+
+
+
+      <div class="row container">
+        <div class="col">
+             <div class="progress">
+                <div class="progress-bar ${(data.status === "RUNNING" ? "bg-warning" : (data.status === "FAILED" ? "bg-danger" : (data.status === "DONE" ? "bg-success" : "") ))}" role="progressbar" style="width: ${(data.progress+k)}%" aria-valuenow="${(data.progress+k)}" aria-valuemin="0" aria-valuemax="100">
+                    ${(data.progress+k)} %
+                </div>
+             </div>
+        </div>
+      </div>
+
+
+		<div class="row container">
+			 <div class="col">
+				<div class="collapse multi-collapse" id="query${queryCounter}">
+				   <div class="card card-body">
+					 <pre> ${(data.query ? JSON.stringify(data.query, undefined, 2) : '')}  </pre>
+				   </div>
+				</div>
+			 </div>
+			 <div class="col">
+				<div class="collapse multi-collapse" id="error${queryCounter}">
+				   <div class="card card-body">
+					  <pre>  ${(data.error ? JSON.stringify(data.error, undefined, 2) : '')}  </pre>
+				   </div>
+				</div>
+			 </div>
+		</div>
+</div>
+</div>
+   </td>
+</tr>
+
+
+
+    `
+    }
+    </#noparse>
+</script>
+<h1> Queries </h1>
+
+
+
+<div id="accordion">
+  <div class="card">
+    <div class="card-header" id="headingOne">
+      <h5 class="mb-0">
+        <button class="btn btn-link" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+          Running <span class="badge badge-warning" id="runningNbr">0</span>
+        </button>
+      </h5>
+    </div>
+
+    <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#accordion">
+
+
+      <div class="card-body">
+
+           <table id="runningQueriesTable">
+                    <tr>
+                       <th>#</th>
+                    </tr>
+           </table>
+      </div>
+    </div>
+  </div>
+  <div class="card">
+    <div class="card-header" id="headingTwo">
+      <h5 class="mb-0">
+        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+          Not started <span class="badge badge-light" id="notStartedNbr">0</span>
+        </button>
+      </h5>
+    </div>
+    <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+      <div class="card-body">
+
+                 <table id="notStartedQueriesTable">
+                          <tr>
+                             <th>#</th>
+                          </tr>
+                 </table>
```

Comment:
Sind Tables hier notwendig? Würden auch divs gehen?

Reply:
ja denn mit divs sieht das nicht so schön aus, hatte ich schon ausprobiert und außerdem ist das Entfernen der Zeilen in Tabelle einfacher als in Divs

Reply:
Wenn du einfach den ganzen inner-body des divs/tables abschießt ist es vermutlich egal. bei div hast du mehr Kontrolle.